### PR TITLE
Allow admin to call runSQL, fix event listening

### PR DESCRIPTION
**Pull request by joewagner**
Two changes here:
1. As a stop-gap, allow the account that deployed the TablelandTables contract to call `runSQL` on behalf of any other account.
2. Fix/Workaround for an issue with how Hardhat emits events.

### CODE REVIEW

File: contracts/TablelandTables.sol [hunk 1]
```diff
@@ -69,7 +69,8 @@ contract TablelandTables is
     );
 
     function runSQL(address caller, uint256 tableId, string memory statement) public {
-        require(caller == _msgSender(), "Tables: caller must be sender"); // temp, caller must be sender (later msg.sender could be a delegate)
+        // temp, caller must be sender or admin (later msg.sender could be a delegate)
+        require(caller != _msgSender() && !hasRole(DEFAULT_ADMIN_ROLE, _msgSender()), "Tables: caller must be sender or admin");
```
Comment (joewagner) [inline contracts/TablelandTables.sol:73]: Simple check to see if the caller is the sender, or if the sender has DEFAULT_ADMIN_ROLE.
This change allows the end to end tests to work, and development of the chess game to move forward.
Also related to https://www.notion.so/textile/Implement-on-behalf-signing-for-relayed-runSQL-in-the-validator-5db1097e259e4726846b95100c6ee71c

File: hardhat.config.ts [hunk 2]
```diff
@@ -35,6 +35,12 @@ const config: HardhatUserConfig = {
       accounts:
         process.env.PRIVATE_KEY !== undefined ? [process.env.PRIVATE_KEY] : [],
     },
+    hardhat: {
+      mining: {
```
Comment (joewagner) [inline hardhat.config.ts:39]: Turning on more traditional mining for the hardhat node allows the Validator to listen for to event logs.

Comment (sanderpick) [reply hardhat.config.ts:39]: nice find

File: contracts/TablelandTables.sol (hunk 1 above)
Comment (sanderpick) [reply contracts/TablelandTables.sol:73]: 👍 thanks for adding this!

**Review by sanderpick — APPROVED** [review]

**Pull request merged**